pos_account_report_invoice_paid/models/invoice.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2020-Today: Druidoo (<http://www.druidoo.io/>)
+# @author: Druidoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    @api.model
+    def _get_pos_details(self):
+        pos_order = self.env['pos.order'].search([(
+            'invoice_id', '=', self.id)])
+        return pos_order
```

Comment:
I'm wondering if adding a new One2many field would work:

```py
# Using One2many as Many2one mirror field
pos_order_id = fields.One2many(
    'pos.order',
    'invoice_id',
    string='POS Order',
    ondelete='set null',
    readonly=True,
)
```

It's like a reverse-many2one.